DFE-Digital/early-years-foundation-reform
# Hfeyp 653 asset folder components

## Ticket and context

Ticket: [HFEYP-653](https://dfedigital.atlassian.net/browse/HFEYP-653)
## Tech review

- Added a column to the content asset model called 'page_name'
- Added radio buttons to form to edit asset page to set correct page name
- Added accordion to asset page, which displays a section for each content page title
- Added a copy link button to each image

### Code quality checks

## Product review
### Is there anything that the product reviewer should know?
The design hasn't been followed exactly for a few reasons:
- the prototype is quite old
- the govuk accordion design has changed
- some radio buttons for selection needed to be added to the edit page to make it make sense

### How can someone see it in review app?
1. Click [the link to review app](https://eyfs-cms-review-pr-586.london.cloudapps.digital/admin/assets)
2. Add an asset. Select which accordion section it should be in by selecting the 'page title'.
3. Verify that it is in the correct section.
4. Edit the page title and verify that it moves to the correct section
